bridge.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import sys
+import tomli
+import re
+
+import libroll as lib
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Bridge eth from L1 to L2.')
+    parser.add_argument('--private-key', type=str, default=None,
+                        help='Private key for the transaction.')
+    parser.add_argument('--account', type=str, default=None,
+                        help='Account to send to.')
+    parser.add_argument('--amount', type=str, default='1ether',
+                        help='Amount of eth to bridge.')
+    parser.add_argument('--name', type=str, default='rollup',
+                        help='Name of the deployment.')
+    parser.add_argument('--config', type=str, default='config.toml',
+                        help='Path to the config file.')
+    args = parser.parse_args()
+
+    config = tomli.load(open(args.config, 'rb'))
+
+    private_key = args.private_key or config["contract_deployer_key"]
+    account = args.account or config["contract_deployer_account"]
+
+    bridge_proxy = json.load(
+        open(os.path.join('deployments', args.name, 'artifacts', 'addresses.json')))['L1StandardBridgeProxy']
```

Comment:
You can use `config.addresses_path` for this.

Reply:
Seems like there is a difference between config_file from config.toml and config from the "Config" object in the generated in deployments/** that I overlooked. I propose (if I may) to leave it like this in this PR. 
I'm pretty sure this will resolve itself when I start moving the bridge script into a subcommand since I'll have access to config_file and the Config object directly.

Reply:
True, I only realized later!